src/recensio/plone/browser/review.py

```diff
@@ -2,6 +2,7 @@
 from html import escape
 from plone import api
 from plone.dexterity.utils import iterSchemata
+from plone.memoize.view import memoize
```

Comment:
This could be cached longer than only for the request.
@ale-rt should I‌ use a different memoizer here?

Reply:
The one you used looks good enough to me